Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Cursors/BaseCursor.cs

```diff
@@ -287,11 +284,15 @@ public virtual void OnBeforeFocusChange(FocusEventData eventData)
         /// <inheritdoc />
         public virtual void OnPointerDown(MixedRealityPointerEventData eventData)
         {
-            foreach (var sourcePointer in eventData.InputSource.Pointers)
+            if (Pointer != null)
             {
-                if (sourcePointer.PointerId == Pointer.PointerId)
+                foreach (var sourcePointer in eventData.InputSource.Pointers)
```

Comment:
nit: consistency: most other loops in this file use for instead of foreach

Reply:
I think this might be fine, since this line was present in the original file:
https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5374/files#diff-89d7de97a7cae3c9e2c96ea7bc566646L290

I'm good either way though.

Reply:
Either way works for me too - just let me know what is preferred.